README.md

```diff
@@ -40,7 +40,7 @@ Once the record is set, you can call the service again to verify that the challe
 has been completed and start issuing domain keys.
 
 ```bash
-curl -F domain=example.com -F domainkey=f4a5cb7f-adac-450c-919f-a12b13cec116 https://eynvwoxb7l.execute-api.us-east-1.amazonaws.com/helix-services/domainkey-provider/v1/
+curl domainkey=f4a5cb7f-adac-450c-919f-a12b13cec116 "https://eynvwoxb7l.execute-api.us-east-1.amazonaws.com/helix-services/domainkey-provider/v1/?domain=example.com&domainkey=f4a5cb7f-adac-450c-919f-a12b13cec116"
```

Comment:
```suggestion
curl -F domainkey=f4a5cb7f-adac-450c-919f-a12b13cec116 -F domain=example.com "https://eynvwoxb7l.execute-api.us-east-1.amazonaws.com/helix-services/domainkey-provider/v1/"
```

Reply:
We should keep this as a POST request, so that no sensitive data is passed via a GET request (this would show up in logs)